module/Company/view/company/admin/edit-job.phtml

```diff
@@ -174,6 +174,19 @@ $element->setAttribute('class', 'form-control');
 <?php endforeach;?>
 </div>
 
+<?php
+$element = $form->get('labels');
+?>
+<div class="form-group<?= count($element->getMessages()) > 0 ? ' has-error' : '' ?>">
+    <label for="<?= $element->getName() ?>" class="control-label col-sm-2">
```

Comment:
This should be replaced with the `BootstrapElementError` Helper similar to the other (already changed) instances.

Reply:
I think it's nice to have it already in place. Even if there's no real form validation at this point in time.